.github/workflows/publish-containers.yaml

```
@@ -201,17 +230,70 @@ jobs:
+                SOURCES="${SOURCES} docker://${FULL_IMAGE}@${digest}"
+              done
+
+              for tag in ${IMAGE_TAGS}; do
+                echo "Creating manifest list for ${FULL_IMAGE}:${tag}"
+                buildah manifest create "${FULL_IMAGE}:${tag}" ${SOURCES}
+                buildah manifest push --all "${FULL_IMAGE}:${tag}" "docker://${FULL_IMAGE}:${tag}"
+                buildah manifest rm "${FULL_IMAGE}:${tag}"
+              done
+            done
+          done
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)